src/Exceptions/UnsupportedAdapterException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Jobilla\CloudNative\Laravel\Exceptions;
+
+class UnsupportedAdapterException extends \Exception
```

Comment:
If empty, you can remove it

Reply:
What do you mean? This exception is used